ctapipe/io/hessio.py

```diff
@@ -91,178 +43,12 @@ def hessio_event_source(url, max_events=None, allowed_tels=None,
     use_event_id : bool
         If True ,'requested_event' now seeks for a particular event id instead
         of index
-    """
-
-    warnings.warn("hessio_event_source has been deprecated. "
```

Comment:
I would keep this warning, we do want this function to be transitioned away from

Reply:
I'll add an `@deprecated` decorator, as with other functions (actually that reminds me that we should define a ctapipe deprecated decorator, instead of using the astropy one, since the message is confusing when it is called an `AstropyDeprecationWarning`